core/src/main/java/io/jenkins/pluginhealth/scoring/model/updatecenter/UpdateCenter.java

```diff
@@ -29,6 +29,8 @@
 
 public record UpdateCenter(Map<String, Plugin> plugins,
                            Map<String, Deprecation> deprecations,
-                           List<SecurityWarning> warnings
+                           List<SecurityWarning> warnings,
+                           List<Map<String, String>> issueTrackers
+
```

Comment:
This does not exists.
Please see https://updates.jenkins.io/current/update-center.actual.json.

The issue trackers are listed within each plugin.

Reply:
For this I need to call the `UpdateCenterPluginPublicationProbe`? 

if it succeeds go ahead?

Reply:
And remove this `List<Map<String, String>> issueTrackers`?

Reply:
you need to get the plugin from the update-center list and from it get its issue trackers list.